src/AdoNet/Orleans.Clustering.AdoNet/Microsoft.Orleans.Clustering.AdoNet.targets

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="4.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Target Name="MicrosoftOrleansClusteringAdoNetCopySqlFiles" BeforeTargets="PrepareForBuild">
+    <!-- These are the source content files needed in the project. Note that $(TargetFramework) cannot be used here since it can be netcoreapp2.0 or somesuch. -->
+    <ItemGroup>
+      <MicrosoftOrleansClusteringAdoNetItems Include="$(MSBuildThisFileDirectory)..\..\lib\netstandard2.0\**\*.sql" />
+    </ItemGroup>
+
+    <!-- The user may have modified the *.sql files fit for deployment. So they shouldn't be overwritten when Nuget restore is made. -->
+    <Copy
```

Comment:
These files are only copied to the project directory, but not added to the project content. The purpose is to make them more findable after Nuget restore. It's not clear what should be done after the files have been brought to the project:

1. The user may want to use some tool to create the database explicitly.
2. The user may want to copy the files to output and distribute with the silo host so the files are available if the database is deployed (or checked if it's deployed) upon start-up.

Including the files as content and marking them as build output is supposedly harmless, albeit might distribute files needlessly. The user may forget/doesn't think to distribute the files if they aren't added automatically and have a slightly worse deployment experience (this is an issue supposedly when one is "kicking the tyres", since running a production deployment one needs to do some QA anyhow). This can be changed also when there's more configuration surface available in the .NET libraries.